composer.json

```diff
@@ -28,15 +28,15 @@
     }
   ],
   "require": {
-    "php": ">=7.0.8",
     "ckeditor/panelbutton": "^4.10",
     "cweagans/composer-patches": "^1.6.5",
-    "drupal-composer/drupal-scaffold": "^2.5",
     "drupal/better_normalizers": "^1.0@beta",
     "drupal/block_content_permissions": "^1.8",
     "drupal/components": "^1",
     "drupal/config_installer": "^1",
-    "drupal/core-recommended": "^8.8.0",
+    "drupal/core-composer-scaffold": "^8.8",
```

Comment:
Please keep drupal-composer/drupal-scaffold we can switch to drupal/core-composer-scaffold after https://www.drupal.org/project/drupal/issues/3091285 gets fixed